docker/ubuntu-18.04.dockerfile-todo

```diff
@@ -18,9 +18,10 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN <<_DEPS
 #!/bin/bash
 apt-get update -y
-apt-get install -y --no-install-recommends \
-  software-properties-common=0.96.24.32.18
-add-apt-repository ppa:ubuntu-toolchain-r/test
+apt-get install software-properties-common -y
+add-apt-repository ppa:ubuntu-toolchain-r/test -y
+add-apt-repository ppa:savoury1/boost-defaults-1.74 -y
+apt-get update -y
```

Comment:
Even though this file is WIP, Please keep the existing formatting here. How you did it in the 20.04 file is correct.